jgrapht-core/src/test/java/org/jgrapht/alg/partition/BipartitePartitionFinderTest.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * (C) Copyright 2018-2018, by Alexandru Valeanu and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht.alg.partition;
+
+import org.jgrapht.Graph;
+import org.jgrapht.generate.*;
+import org.jgrapht.graph.DefaultEdge;
+import org.jgrapht.graph.SimpleGraph;
+import org.jgrapht.util.SupplierUtil;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Random;
+
+/**
+ * Tests for {@link BipartitePartitionFinder}
+ *
+ * @author Alexandru Valeanu
+ */
+public class BipartitePartitionFinderTest {
```

Comment:
There are tests for the bipartite code in the class `GraphTestsTest`.

Reply:
Since I won't remove the two methods from `GraphTests` (doing so would probably break some code) I think that we can keep the three tests for those methods in `GraphTestsTest`. What do you think about this?

Reply:
It's better to keep the tests as close to the actual implementation as possible. So better to move the relevant tests from GraphTestsTest to here.